models/pardot/model.sql

```diff
@@ -81,3 +81,24 @@ create or replace view pardot_analysis.visitoractivity_events_meta as (
   select 18, '', 'reopened opportunity'
 
 );
+
+
+create or replace view {schema}.visitoractivity as (
+  --this table has a bunch of types that really should be event actions but are very poorly formulated.
+  --the custom logic in this view is an attempt to fix that.
+  --not all of the various type / type_name combinations have been accounted for yet; I still need to determine exactly what some of them mean.
+  select
+    -- event_stream interface
+    va.created_at       as "@timestamp",
+    t.type_decoded      as "@event",
+    va.prospect_id      as "@user_id",
+    va.*
+  from
+    olga_pardot.visitoractivity va
+    inner join {schema}.visitoractivity_events_meta e
+      on va."type" = e."type" and va.type_name = e.type_name
+    inner join {schema}.visitoractivity_types_meta t
+      on va."type" = t."type"
+);
+
+COMMENT ON VIEW {schema}.visitoractivity IS 'timeseries,funnel,cohort';
```

Comment:
but I don't think switching to this approach is high priority - if this is working lets leave it for now